src/site/content/en/blog/catalina-system-ui/index.md

```diff
@@ -0,0 +1,246 @@
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_grad_v2.mp4">
+  </video>
+</figure>
+
+### `YAXS`
+Stretches the glyph vertically. It accepts values between `400` and `1000`.
+
+```css
+/* 400-1000 */
+font-variation-settings: 'YAXS' 500;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_yaxs_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_yaxs_v2.mp4">
+  </video>
+</figure>
+
+### Combining the options
+With a few lines of CSS, we can tweak the font settings into a bold of our choice or try out other interesting combinations:
+
+```css
+font-weight: 700;
+font-weight: bold;
+font-variation-settings: 'wght' 750, 'YAXS' 600, 'GRAD' 500, 'opsz' 20;
+```
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_combined_v2.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/macos-system-ui/system-ui_combined_v2.mp4">
+  </video>
+</figure>
+
+And just like that, Chromium users on macOS see your upgraded, custom 750 weight with some fun other tweaks 👍 
+
+## Sandbox
+
+<div class="glitch-embed-wrap" style="height: 480px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/macos-system-ui?path=style.css&attributionHidden=true"
+    alt="macOS Catalina system-ui sandbox on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+macOS 10.15 added new features to its system font, and in macOS 10.15 a tricky `system-ui` bug was logged in the Chromium bug tracker. I wonder if they are related!?
+
+{% Banner 'neutral' %}The image for this article was made with CSS and system-ui, here's the [codepen](https://codepen.io/argyleink/pen/eYpbwWd){% endBanner %}
+
+## Appendix: The 'system-ui' regression {: #regression }
+
+{% Aside %}
+The rest of this blog post explains how `system-ui` broke in Chromium 80 and how the Chromium engineers fixed it. If this isn't interesting or relevant for you, you can stop reading now.
+{% endAside %}
+
+This story starts with a different bug: [#1005969](https://crbug.com/1005969). This was reported against macOS 10.15 because the system-ui font spacing looked narrow and crammed.
+
+<figure class="w-figure">
+  <img src="./tight-chrome-spacing.png" alt="A comparison of two paragraphs from a Facebook group page. On the left is Chrome and the right is Safari, and Chrome is subtle but slightly tighter in spacing">
+  <figcaption class="w-figcaption">Chrome on left (tighter kerning), Safari on right (better optical spacing)</figcaption>
+</figure>
```

Comment:
Much better example screenshot. And a clever way to avoid the controversial content. 👍